src/apps/interactions/client/transformers.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable camelcase */
+const { get } = require('lodash')
+
+const { format } = require('../../../client/utils/date-utils')
+const urls = require('../../../lib/urls')
+
+const formatContacts = (contacts) =>
+  contacts.length > 1
+    ? 'Multiple contacts'
+    : contacts.map((contact) => getContactName(contact))
+
+const getContactName = (contact) =>
+  get(contact, 'name') ? contact.name : 'Unknown contact'
+
+const formatParticipantName = (participant) =>
+  get(participant, 'adviser') && get(participant, 'team')
+    ? `${participant.adviser.name}, ${participant.team.name}`
+    : get(participant, 'adviser')
+    ? participant.adviser.name
+    : 'Unknown adviser'
+
+const formatParticipants = (dit_participants) =>
+  dit_participants.length > 1
+    ? 'Multiple advisers'
+    : dit_participants.map((participant) => formatParticipantName(participant))
+
+const getbadgeLabel = (type, hasFeedback = false) => {
+  const badges = {
+    interaction: 'interaction',
```

Comment:
Looking at the screenshots, think this should be a capital Interaction, or else the badge is in lower case.
```suggestion
    interaction: 'Interaction',
```

Reply:
Good spot thanks 👍